apps/src/templates/CodeWorkspace.jsx

```diff
@@ -84,7 +84,7 @@ class CodeWorkspace extends React.Component {
     if (textbox.style.bottom) {
       $(textbox).animate(
         {bottom: debuggerHeight},
-        {step: utils.fireResizeEvent}
+        {done: utils.fireResizeEvent}
```

Comment:
Fixes a long-standing issue where the code workspace would resize to the wrong size. This is especially apparent on computers that are close to their maximum ram when running applab. This was fixed as part of this PR because the droplet changes exacerbated the issue. I checked with @ryansloan, and he agrees that the new UI is preferred.

**OLD**
![workspaceRendering](https://user-images.githubusercontent.com/8324574/61733121-f3660e80-ad33-11e9-9049-40c386ae535e.gif)

**NEW**
![workspaceRenderingNew](https://user-images.githubusercontent.com/8324574/61733262-43dd6c00-ad34-11e9-8ae2-bf66e3cd6430.gif)

Reply:
Awesome!